libs/platform/src/lib/components/link/link.component.html

```diff
@@ -0,0 +1,16 @@
+<a
+    #link
+    class="fd-link"
+    [attr.id]="id"
+    [attr.href]="href"
+    [target]="target"
+    [attr.aria-disabled]="disabled"
+    [ngClass]="{ 'fd-link--emphasized': isEmphasized, 'fd-link--inverted': inverted, 'fdp-link-truncate__txt': !wrap }"
+    (click)="clicked($event)"
+    [ngStyle]="{ width: width }"
```

Comment:
Why do we need width? fiori spec does not say anything like this. plus there is no truncation.

Reply:
removed with, width should be coming from parent element.